### PR TITLE
Improve Generic Information and Housekeeping sections

### PR DESCRIPTION
**Pull request by fviale**
- add popular generic information (START_AT, NODESOURCENAME, etc) reference
 - add the new generic information REMOVE_DELAY
 - update housekeeping section
 - fix small mistakes in title levels.

### CODE REVIEW

File: src/docs/user/references/GenericInformation.adoc [hunk 1]
```diff
@@ -22,10 +22,63 @@ For example:
 </task>
 ----
 
-=====  Icon management variables
+Below are several examples of Generic information:
```
Comment (mcompastie) [inline src/docs/user/references/GenericInformation.adoc:25]: Several examples of Generic Information are presented below:

File: src/docs/user/references/GenericInformation.adoc [hunk 2]
```diff
@@ -22,10 +22,63 @@ For example:
...
+`NODESOURCENAME` is used to deploy workflow tasks in a Node Source controlled by a *Dynamic Policy*.
+
+See <<../admin/ProActiveAdminGuide.adoc#_dynamic_policy,Dynamic Policy>> for more information.
+
+`NODESOURCENAME` can be defined at *job-level* (applies to all tasks of a workflow) or at  *task-level* (applies to a single task).
+
+==== Documentation
+
+The `Documentation` generic information allows to associate an html documentation with a workflow.
+Its value must contain an url pointing to the workflow documentation.
```
Comment (mcompastie) [inline src/docs/user/references/GenericInformation.adoc:73]: URL

File: src/docs/user/references/GenericInformation.adoc [hunk 3]
```diff
@@ -22,10 +22,63 @@ For example:
...
+`START_AT` can be defined at *job-level* (delay the execution of the whole job) or at  *task-level* (delay the execution of a single task).
+
+
+==== REMOVE_DELAY
+
+The `REMOVE_DELAY` Generic Information can be used to control when a job is removed from the scheduler database after its termination.
+
+The <<../admin/ProActiveAdminGuide.adoc#_housekeeping,housekeeping mechanism>> must be configured to allow usage of `REMOVE_DELAY`.
+
+`REMOVE_DELAY` overrides the global `pa.scheduler.core.automaticremovejobdelay` setting for a particular job. It allows a job to be removed either *before* or *after* the globally-configured delay.
```
Comment (mcompastie) [inline src/docs/user/references/GenericInformation.adoc:46]: `the globally-configured delay.` is confusing. I am not sure to understand how it is related to the `pa.scheduler.core.automaticremovejobdelay` configuration key.

File: src/docs/user/references/GenericInformation.adoc [hunk 4]
```diff
@@ -22,10 +22,63 @@ For example:
...
+
+The general format of the `REMOVE_DELAY` generic information is `[hh:mm:ss]`, where h is hour, m is minute and s is second.
```
Comment (nebilbenmabrouk) [inline src/docs/user/references/GenericInformation.adoc:48]: What about the case when the number of hours exceeds two digits? For example 720:00:00, which means within a month (24X30)

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 5]
```diff
@@ -2525,20 +2526,25 @@ This mechanism has two phases:
...
+TIP: Additionnally, it is possible to configure a remove delay for individual job execution using the `REMOVE_DELAY` <<../user/ProActiveUserGuide.adoc#_remove_delay,generic information>>.
 
-The following parameters are accessible in <<Scheduler Properties,the scheduler config file>>.
-One can edit the `pa.scheduler.core.automaticremovejobdelay` parameter to set the delay before
-marking a job to be removed.
-The `pa.scheduler.core.removejobdelay` parameter can be used in the case where the delay should only
-start once the job result has been accessed.
+If the `pa.scheduler.core.automaticremovejobdelay` setting is set to `0` (the default), the Housekeeping mechanism is *disabled* (no occurrence of housekeeping).
+
+IMPORTANT: Please note that housekeeping mechanism is not applied to the jobs
```
Comment (nebilbenmabrouk) [inline src/docs/admin/ProActiveAdminGuide.adoc:2538]: that **the** housekeeping mechanism

File: src/docs/user/references/GenericInformation.adoc (hunk 4 above)
Comment (fviale) [reply src/docs/user/references/GenericInformation.adoc:48]: I agree

File: src/docs/user/references/GenericInformation.adoc [hunk 6]
```diff
@@ -4,7 +4,7 @@ In addition to variables, another key/value structure can be accessed inside a s
 Generic information semantics differ from <<_job_variables>> semantics in the following way:
 
 * Generic information can be accessed inside a script, but cannot be modified.
-* Generic information can be defined at *job level* or at *task level*. If the same generic information is defined at job level and at task level, the latter value takes precedence inside the task scope.
+* Generic information can be defined at *job-level* or at *task-level*. If the same generic information is defined at job level and at task level, the latter value takes precedence inside the task scope.
```
[the remark this answers is not in the document]
Comment (fviale) [reply src/docs/user/references/GenericInformation.adoc:7]: ok

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 7]
```diff
@@ -2525,20 +2526,25 @@ This mechanism has two phases:
 . Once a job is finished, it sets its scheduled time for removal (the job expiration date)
 . Actual cleaning of expired jobs from the scheduler and/or the database will be periodically triggered and performed in a bulk operation
 
-Please note that housekeeping mechanism is not applied to the jobs
-which were already in the database when housekeeping was switched on.
+To configure housekeeping, the following parameters are accessible in <<Scheduler Properties,the scheduler config file>>:
```
[the remark this answers is not in the document]
Comment (fviale) [reply src/docs/admin/ProActiveAdminGuide.adoc:2529]: ok

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 8]
```diff
@@ -2525,20 +2526,25 @@ This mechanism has two phases:
...
+
+ * `pa.scheduler.core.automaticremovejobdelay` : delay in seconds after job termination to remove the job.
+ * `pa.scheduler.core.removejobdelay` : instead of using job termination time to initiate the delay, use the result retrieval time (for example, when a user opens the job results in the scheduler portal).
```
[the remark this answers is not in the document]
Comment (fviale) [reply src/docs/admin/ProActiveAdminGuide.adoc:2532]: ok

**Pull request merged**